helm/prometheus-rules/templates/platform/atlas/alerting-rules/mimir.rules.yml

```diff
@@ -151,6 +151,27 @@ spec:
         severity: page
         team: atlas
         topic: observability
+    - alert: MimirRulerTooManyFailedQueries
+      annotations:
+        dashboard: 631e15d5d85afb2ca8e35d62984eeaa0/mimir-ruler
+        description: '{{`Mimir Ruler {{ $labels.pod }} is experiencing {{ printf "%.2f" $value }}% errors while evaluating rules.`}}'
+        opsrecipe: mimir/
+      expr: |
+        100 * (
+        sum by (installation, cluster_id, pipeline, provider, namespace, pod) (rate(cortex_ruler_queries_failed_total[1m]))
```

Comment:
If a customer installs mimir on his WC, this alert will monitor it.
Maybe we want to add a `cluster-type=management-cluster` filter?

Reply:
Isn't this the case for all our rules here ?

Reply:
Maybe.
The thing is no customer has any mimir installed yet, but recently I created Loki alerts where I had to add this filter.